### DISCUSSION TITLE
facet grid titles with sequential identifiers

### DISCUSSION THREAD

**Discussion by klapo** [mwaskom/seaborn#3481] [Q&A] [answered]
Hello seaborn community,

I would like to label my facet grid subplots sequentially (e.g., 'a)', 'b)' 'c)' etc in the titles before the column name). Is there a way to use the `set_titles` template to do so? Or do I need to manually iterate through my subplots and add the sequential identifier by hand?

Thank you!

**Comment by mwaskom** [comment, marked as answer]
I think iteration is the way to go here. It's maybe technically possible to hack the formatting template with a string subclass to do it "automatically" but that seems more complicated than a two line for-loop.

Python isn't R, you're allowed to iterate over here :)

**Reply by klapo** [reply]
Thanks for the reply! I assumed that was the case and had it implemented, but wanted to know if there was a better generalization.
